Testscripts/Linux/xfstesting.sh

```diff
@@ -237,15 +251,16 @@ Main() {
             # Azure provides links in /dev/disk/azure/ to /dev/sd[a-z] devices
             # this is needed as some 5+ kernels do not assign drive letters in any particular order
             # (previously the test disk was /dev/sdc)
-            testDisk=$(readlink -f /dev/disk/azure/scsi1/* | sed "s@/dev/@@g" | head -1)
+            #testDisk=$(readlink -f /dev/disk/azure/scsi1/* | sed "s@/dev/@@g" | head -1)
+            testDisk="sdc"
             ConfigureDisks "${testDisk}" "${testDisk}1" "${testDisk}2" "$FSTYP" "test" "scratch"
             # Configure disk in env and xfstests config file
             TEST_DEV="/dev/${testDisk}1"
             echo "TEST_DEV=${TEST_DEV}" >> ${XFSTestConfigFile}
         fi
     fi
     # Copy config file into the xfstests folder
-    dos2unix ${XFSTestConfigFile}
+    sed -i 's/\r$//' ${XFSTestConfigFile}
```

Comment:
have we checked that dos2unix still absent in mariner?

Reply:
Reverted to previous commit.